### PR TITLE
Fix issues in strings file

### PR DESCRIPTION
**Pull request by flodolo**
– Fix strings
- Use … and ’ in strings
- Remove extra trailing spaces in comments and strings
- Consistency in comment style (closing period)

### CODE REVIEW

File: src/_locales/en/messages.json [hunk 1]
```diff
@@ -99,86 +99,86 @@
...
+    "description": "Header if a VPN installation was not found."
   },
   "bodyOpenMsg": {
     "message": "To use the Mozilla VPN Firefox extension, please open the Mozilla VPN app first.",
-    "description": "Popup subtitle text"
+    "description": "Popup subtitle text."
   },
   "bodyInstallMsg": {
-    "message": "Note that Mozilla VPN is a paid subscription service."
+    "message": "To use this extension, first download and install the Mozilla VPN app on your computer."
```
Comment (flodolo) [inline src/_locales/en/messages.json:163]: I believe this is the correct string based on Figma. Right now it's identical to the footer.

File: src/_locales/en/messages.json [hunk 2]
```diff
@@ -188,107 +188,107 @@
...
   },
   "onboarding2_title": {
     "message": "See different websites from different locations",
-    "description": "Title in onboarding panel"
+    "description": "Title in onboarding panel."
   },
   "onboarding2_body": {
-    "message": "",
-    "description": "Body text in onboarding panel"
+    "message": "News from London. Sports from Sao Paulo. Shopping from Tokyo. Set website-specific locations for personalized browsing.",
```
Comment (flodolo) [inline src/_locales/en/messages.json:222]: I assume this should be the content based on Figma

**Review by flodolo — COMMENTED** [review]
Worth noting that I didn't compare all the strings with Figma, and it migth be worth doing. I've only spotted a couple that look wrong.

**Review by strseb — APPROVED** [review]

File: src/_locales/en/messages.json (hunk 1 above)
Comment (lesleyjanenorton) [reply src/_locales/en/messages.json:163]: Thank you!

File: src/_locales/en/messages.json (hunk 2 above)
Comment (lesleyjanenorton) [reply src/_locales/en/messages.json:222]: Thank you!

**Review by lesleyjanenorton — APPROVED** [review]

**Pull request merged**